src/frontend/src/optimizer/logical_optimization.rs

```diff
@@ -236,11 +236,17 @@ lazy_static! {
         ApplyOrder::TopDown,
     );
 
-    static ref AGG_ON_INDEX: OptimizationStage = OptimizationStage::new(
+    static ref TOP_N_ON_INDEX: OptimizationStage = OptimizationStage::new(
         "Agg on Index",
         vec![TopNOnIndexRule::create()],
         ApplyOrder::TopDown,
     );
+
+    static ref MIN_MAX_ON_INDEX: OptimizationStage = OptimizationStage::new(
+        "Min/Max on Index",
+        vec![MinMaxOnIndexRule::create()],
+        ApplyOrder::TopDown,
+    );
```

Comment:
Could you please merge these two stages? I intend to group rules that serve a similar function together.

Reply:
Thanks for pointing this out!